ha/core/controllers/pcs/pcs_controller.py

```diff
@@ -24,4 +32,66 @@ def __init__(self):
         """
         Initalize pcs controller
         """
-        super(PcsController, self).__init__()
\ No newline at end of file
+        super(PcsController, self).__init__()
+        self._execute = SimpleCommand()
+
+    def check_resource_failcount(self) -> bool:
```

Comment:
Name of function is check_resource_failcount while it's also doing cleanup. This should be two differen function.

def check_resource_failcount():
      pass

def clean_failure_count():
      pass

def heal_resource():
      if check_failure_count():
          clean_failure_count()

Reply:
Done